traffic_portal/test/integration/specs/Profiles.spec.ts

```diff
@@ -29,49 +29,70 @@ const loginPage = new LoginPage();
 const topNavigation = new TopNavigationPage();
 const profilesPage = new ProfilesPage();
 
-describe('Setup API for Profiles', () => {
-    it('Setup', async () => {
-        await api.UseAPI(profiles.setup);
-    });
-});
+beforeAll(async function () {
+    await api.UseAPI(profiles.setup);
+})
+
```

Comment:
So you don't wanna do this, because putting a `beforeAll` in the global scope like that is going to run this before every single `it` in the entire integration testing suite.

Reply:
I see that's why the test took so long. I think I will add a 1 beforeAll in the config before all the test run.

Reply:
Could be, hard to say.

What would that `beforeAll` in the config module be doing? It shouldn't navigate to the Profiles page, that'll have the same problem.